### ISSUE TITLE
I open setting cropping But I want to hidden crop frame in open camera and Show on choose image from library

### ISSUE THREAD

**Issue by rrakchon** [AlexLittlejohn/ALCameraViewController#228]

**Comment by pepejeria** [comment]
Do you mean that the crop frame should be hidden when you wanna take a picture? But shown after selecting an image from the library? If yes, then this would be a nice feature to have. Unless there is some way to achieve this with the current version.

**Comment by pepejeria** [comment]
In CameraViewController.swift, line 173, change that line to 
`cameraOverlay.isHidden = true`

to achieve this. Is there any special reason why this should be shown when taking a picture?

**Comment by theolof** [comment]
Yeah, I agree. It shouldn't be visible when taking the picture, especially since it isn't used until the next view. Now it's a bit confusing for the user.

Another solution would to be to only capture the photo in the crop frame when taking the picture.

**Comment by MZC0829** [comment]
+1
